data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/damage/damage_multiply_fraction.cfg

```diff
@@ -8,7 +8,7 @@
 # Have the side 1 leader attack the side 2 leader with both its weapons
 ##
 # Expected end state:
-# The side 1 leader has 2 weapons both of which have 6 damage (3*3, the .5 is truncated)
+# The side 1 leader has 2 weapons both of which have 8 damage (3*3, the .5 is rounded up)
 #####
 #ifndef SCHEMA_VALIDATION
 {COMMON_KEEP_A_B_UNIT_TEST "damage_multiply_fraction" (
```

Comment:
I do not see any test about 9467. This case verifies interaction of value+multiply, but there is no verification of multiply alone.

Reply:
test added, with 4*1.999, after 2 digitd, multiply must be rounded up to value double who be 2

Reply:
That test still specifies value, so it does not cover #9467 which has only multiply without value.

Reply:
changed with no value specified